bazel/benchmark/driver-utilities/BUILD.bazel

```diff
@@ -1,19 +1,26 @@
-load("//bazel:defaults.bzl", "ts_library")
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
 
 package(default_visibility = ["//visibility:public"])
 
-ts_library(
+ts_project(
```

Comment:
I'd suggest we move the driver utilities into FW `modules/`

Reply:
I can do that instead.